network/src/network.rs

```diff
@@ -192,14 +191,7 @@ impl NetworkState {
                 duration.as_secs(),
                 reason
             );
-            metrics!(
-                counter,
-                "ckb.network.ban_peer",
-                1,
-                "peer_addr" => addr.to_string(),
-                "duration" => duration.as_secs().to_string(),
-                "reason" => reason.clone(),
-            );
+            ckb_metrics::CKB_NETWORK_BAN_PEER.inc();
```

Comment:
We may add reason as a label, but let's defer it until we really need this.

Reply:
We can expose `reason` to Prometheus, but it's preferable for `reason` to be an enumerated value known at compile time. This way, `reason` won't occupy too much memory space.

Reply:
It's easy to break, for example, if someone formats a message and adds the peer id into it.